front/cypress/e2e/project_page_pb.cy.ts

```diff
@@ -20,3 +22,77 @@ describe('Continuous PB project', () => {
     cy.get('#e2e-continuos-project-idea-cards');
   });
 });
+
+describe('Budgeting CTA bar', () => {
+  let projectId: string;
+  let projectSlug: string;
+  const projectTitle = randomString();
+  const description = randomString();
+  const ideaTitle = randomString();
+  const ideaContent = Math.random().toString(36);
+  let ideaId: string;
+
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.apiCreateProject({
+      type: 'continuous',
+      title: projectTitle,
+      descriptionPreview: description,
+      description,
+      publicationStatus: 'published',
+      participationMethod: 'budgeting',
+      maxBudget: 100,
+    })
+      .then((project) => {
+        projectId = project.body.data.id;
+        projectSlug = project.body.data.attributes.slug;
+      })
+      .then(() => {
+        return cy.apiCreateIdea(
+          projectId,
+          ideaTitle,
+          ideaContent,
+          undefined,
+          undefined,
+          undefined,
+          21
+        );
+      })
+      .then((idea) => {
+        ideaId = idea.body.data.id;
+      });
+  });
+
+  after(() => {
+    cy.apiRemoveIdea(ideaId);
+    cy.apiRemoveProject(projectId);
+  });
+
+  it('shows the CTA to the user to allocate their budget when the user has not yet participated', () => {
```

Comment:
Should we add a test for when the user has participated?

Reply:
This test covers both scenarios. I can change the wording to reflect that